homeassistant/components/tado/climate.py

```diff
@@ -168,45 +171,46 @@ def create_climate_entity(
 
             supported_hvac_modes.append(TADO_TO_HA_HVAC_MODE_MAP[mode])
             if (
-                capabilities[mode].get("swings")
-                or capabilities[mode].get("verticalSwing")
+                capabilities[mode].get(TADO_SWING_SETTING)
+                or capabilities[mode].get(TADO_VERTICAL_SWING_SETTING)
                 or capabilities[mode].get("horizontalSwing")
```

Comment:
Wouldn't it be better to do
```py
TADO_SWING_SETTING in capabilities[mode]
```
Instead of all the `.get()`?

Reply:
you're right